Feature/Onboarding/Sources/Concreate/RadioButton.swift

```diff
@@ -0,0 +1,110 @@
+//
+//  RadioButton.swift
+//  FeatureLayer
+//
+//  Created by 최재혁 on 7/26/25.
+//
+
+import UIKit
+import DesignSystem
+
+import Then
+import SnapKit
+
+// MARK: - RadioButtonViewDelegate (단일 라디오 버튼이 선택되었음을 알리는 프로토콜)
+protocol RadioButtonViewDelegate: AnyObject {
+    func radioButtonView(_ radioButtonView: RadioButtonView, didSelect isSelected: Bool)
+}
+
+// MARK: - RadioButtonView (개별 라디오 버튼)
+class RadioButtonView: UIView {
+
+    // 델리게이트를 통해 상위 뷰에 선택 상태를 알림
+    weak var delegate: RadioButtonViewDelegate?
+
+    // 라디오 버튼의 바깥 원
+    private let circleView = UIView().then {
+        $0.layer.cornerRadius = 10
+        $0.layer.borderWidth = 1.5
+        $0.layer.borderColor = DesignSystemAsset.Colors.gray7.color.cgColor
+    }
+
+    // 선택되었을 때 나타나는 안쪽 채워진 원
+    private let selectedIndicatorView = UIView().then {
+        $0.layer.cornerRadius = 5
+        $0.backgroundColor = DesignSystemAsset.Colors.white.color
+    }
+
+    // 라디오 버튼 옆의 텍스트 레이블
+    private let titleLabel = UILabel()
+
+    // 라디오 버튼의 선택 상태 (true: 선택됨, false: 선택 안 됨)
+    var isSelected: Bool = false {
+        didSet {
+            // isSelected 값이 변경될 때마다 UI 업데이트
+            if isSelected {
+                circleView.backgroundColor = DesignSystemAsset.Colors.primary1.color
+                circleView.layer.borderColor = DesignSystemAsset.Colors.primary1.color.cgColor
+            } else {
+                circleView.backgroundColor = DesignSystemAsset.Colors.white.color
+                circleView.layer.borderColor = DesignSystemAsset.Colors.gray7.color.cgColor
+            }
+        }
+    }
+
+    // 라디오 버튼에 표시될 텍스트
+    var title: String? {
+        didSet {
+            let style = Typography.Body_14_M
+            style.color = DesignSystemAsset.Colors.black.color
+            titleLabel.attributedText = title?.set(style: style)
+        }
+    }
```

Comment:
특정 뷰 컴포넌트와 동기화되어야 하는 값은 데이터의 원천을 둘로 나누는 것보다 getter, setter 로 titleLabel 의 text 와 엮어주는 게 좋을 것 같아요.
```suggestion
    var title: String? {
        get { titleLabel.text }
        set {
            let style = Typography.Body_14_M
            style.color = DesignSystemAsset.Colors.black.color
            titleLabel.attributedText = newValue?.set(style: style)
        }
    }
```

Reply:
component 부분에서 해당 코드로 작성되 부분들이 있는 것 같아서 다른 부분도 확인 후 수정하겠습니다!